cql3/cql3_type.cc

```diff
@@ -473,27 +473,40 @@ sstring maybe_quote(const sstring& identifier) {
     return result;
 }
 
-sstring quote(const sstring& identifier) {
+template <char C>
+static sstring quote_with(const sstring& str) {
```

Comment:
Please change to a regular parameter. We have enough templates this is slow path code anyway, replacing a parameter with a literal doesn't bring any benefit.